tests/table/test_table_io.py

```diff
@@ -35,11 +35,11 @@
     "integers": [i for i in range(TEST_DATA_LEN)],
     "floats": [float(i) for i in range(TEST_DATA_LEN)],
     "bools": [True for i in range(TEST_DATA_LEN)],
+    "fixed_sized_arrays": [[i for _ in range(4)] for i in range(TEST_DATA_LEN)],
```

Comment:
This adds a test case for fixed-size list arrays, but it looks you added a FFI match case for variable-size list arrays.

Reply:
Ah good catch! Fixing now.

Reply:
Oh actually - this should return you fixed-sized lists of size 4. Should be ok!